src/recipe/file.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { S3Service } from '../recipe/s3.service';
+
+@Injectable()
+export class FileService {
+  constructor(
+    private configService: ConfigService,
+    private s3Service: S3Service,
+  ) {}
+
+  async uploadFileToS3(file: Express.Multer.File) {
+    const bucket = this.configService.getOrThrow('S3_BUCKET_NAME');
```

Comment:
I think the better approach would be use bucket name from envs and add fallback value if it's to present. Egample:

```suggestion
    const bucket = this.configService.get('S3_BUCKET_NAME') || 'default_bucket_name';
```

Reply:
What is default_bucket_name

Reply:
just an example. The idea is that you use bucket name from env variables. If this is not present (this particular env variable was not set in the system) then you hardcode a default value. 
In your case, it should be your bucket name